fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/AbstractMaterializedViewAggregateRule.java

```diff
@@ -426,7 +447,8 @@ private Pair<Set<? extends Expression>, Set<? extends Expression>> topPlanSplitT
                 topGroupByExpressions.add(expression);
             }
         });
-        return Pair.of(topGroupByExpressions, topFunctionExpressions);
+        groupByExpressionSet.removeAll(topGroupByExpressions);
```

Comment:
`topPlanSplitToGroupAndFunction`  means get the group by and agg fuctions expression pair from top plan, this change the measing of the method
May be we can get the result of method `topPlanSplitToGroupAndFunction` and then do your removeAll logic